script/Deploy.s.sol

```diff
@@ -106,6 +109,12 @@ contract DeployGoerli is GoerliParams, Deploy {
   }
 
   function setupEnvironment() public virtual override updateParams {
+    // Deploy oracle factories
+    chainlinkRelayerFactory = new ChainlinkRelayerFactory(OP_GOERLI_CHAINLINK_SEQUENCER_UPTIME_FEED);
+    uniV3RelayerFactory = new UniV3RelayerFactory();
+    denominatedOracleFactory = new DenominatedOracleFactory();
+    delayedOracleFactory = new DelayedOracleFactory();
+
```

Comment:
i understand that this is bc of the sequencer uptime feed is a parameter, i'm ok with it